test/models/backend_api_test.rb

```diff
@@ -23,4 +23,12 @@ def test_default_api_backend
     @backend_api.private_endpoint = 'https://example.org:3/path'
     assert @backend_api.valid?
   end
+
+  test '.orphans should return backend apis that do not belongs to any service' do
+    FactoryBot.create(:backend_api_config)
```

Comment:
I do not get this test :)
If I change the factory it will probably fail

Create a Service, then remove the BackendApiConfig linked to it is sufficent right?

Reply:
Makes sense.. I constructed the backend_api_config to be more explicit, but I agree with you. it is better to create the service only.